compiler/rustc_span/src/span_encoding.rs

```diff
@@ -1,9 +1,3 @@
-// Spans are encoded using 1-bit tag and 2 different encoding formats (one for each tag value).
```

Comment:
This appeared out-of-date to me, but I might be wrong.

Reply:
This comment does not seem out of date to me afaict.

This is a short introduction to how the struct `Span` works and the reference to the internal thread seems aslo relevant.

Can you elaborate on why you assume this comment is out of date?

Reply:
It was just a guess, really. Will put this back.